recipes/puma/meta.yaml

```
@@ -0,0 +1,97 @@
+{% set name = "puma" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/jcfergus/puma-dev/-/archive/v{{ version }}/puma-dev-v{{ version }}.tar.gz
+  sha256: 938737eda88222d1c9191e5ddc1237e7fe306e8d961fd8cce96f0a5dc1d95a53
+
+build:
+  number: 0
+  skip: True  # [win and linux]
```

Think this should be `or` otherwise it won't work.

```suggestion
  skip: True  # [win or linux]
```

---

Thank you, good catch